src/CoreText/CTFontCollection.cs

```diff
@@ -196,13 +196,24 @@ public CTFontCollection (CTFontDescriptor []? queryDescriptors, CTFontCollection
 			return CFArray.ArrayFromHandleFunc (cfArrayRef, fd => new CTFontDescriptor (fd, false), true)!;
 		}
 
+#if NET
+		[DllImport (Constants.CoreTextLibrary)]
+		static unsafe extern IntPtr CTFontCollectionCreateMatchingFontDescriptorsSortedWithCallback (
+				IntPtr collection, delegate* unmanaged<IntPtr, IntPtr, IntPtr, CFIndex> sortCallback, 
+				IntPtr refCon);
+#else
 		[DllImport (Constants.CoreTextLibrary)]
 		static extern IntPtr CTFontCollectionCreateMatchingFontDescriptorsSortedWithCallback (
 				IntPtr collection, CTFontCollectionSortDescriptorsCallback sortCallback, IntPtr refCon);
+#endif
 
 		delegate CFIndex CTFontCollectionSortDescriptorsCallback (IntPtr first, IntPtr second, IntPtr refCon);
```

Comment:
Minor: we don't need this delegate declaration in .NET anymore, so it can be inside `#if !NET`